### PR TITLE
Add f_cdf Presto Function

### PR DESCRIPTION
**Pull request by rrando901**
Add Presto function f_cdf following the Java implementation([source](https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java#L924))

Resolves: https://github.com/facebookincubator/velox/issues/5379

### CODE REVIEW

**Comment by rrando901** [comment]
Fuzzer output

```
I20230623 17:46:42.158910 443986 ExpressionVerifier.cpp:56] ===================
I20230623 17:46:42.158933 443986 ExpressionVerifier.cpp:67] All results match.
I20230623 17:46:42.158983 443986 ExpressionFuzzer.cpp:1357] ==============================> Done with iteration 428
I20230623 17:46:42.159350 443986 ExpressionFuzzer.cpp:1092] ==============================> Top 1 by number of rows processed
I20230623 17:46:42.159369 443986 ExpressionFuzzer.cpp:1094] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230623 17:46:42.159384 443986 ExpressionFuzzer.cpp:1098] f_cdf 44818 100.00% 5376
I20230623 17:46:42.159400 443986 ExpressionFuzzer.cpp:1104] ==============================> Bottom 1 by number of rows processed
I20230623 17:46:42.159415 443986 ExpressionFuzzer.cpp:1106] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230623 17:46:42.159427 443986 ExpressionFuzzer.cpp:1111] f_cdf 44818 100.00% 5376
I20230623 17:46:42.159442 443986 ExpressionFuzzer.cpp:1124] ==============================> All stats sorted by number of times the function was chosen
I20230623 17:46:42.159456 443986 ExpressionFuzzer.cpp:1126] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230623 17:46:42.159468 443986 ExpressionFuzzer.cpp:1130] f_cdf 44818 100.00% 5376
[==========] Running 0 tests from 0 test suites.
[==========] 0 tests from 0 test suites ran. (0 ms total)
[  PASSED  ] 0 tests.
```

File: velox/functions/prestosql/tests/ProbabilityTest.cpp [hunk 1]
```diff
@@ -155,5 +155,52 @@ TEST_F(ProbabilityTest, binomialCDF) {
...
+  const auto fCDF = [&](std::optional<double> df1,
+                        std::optional<double> df2,
+                        std::optional<double> value) {
+    return evaluateOnce<double>("f_cdf(c0, c1, c2)", df1, df2, value);
+  };
+
+  EXPECT_EQ(fCDF(2.0, 5.0, 0.0), 0.0);
+  EXPECT_EQ(fCDF(2.0, 5.0, 0.7988), 0.50001145221750731);
+  EXPECT_EQ(fCDF(2.0, 5.0, 3.7797), 0.89999935988961155);
+  EXPECT_EQ(fCDF(5.02, 10.0, 0.72), 0.37628199518500832);
```
Comment (aditi-pandit) [inline velox/functions/prestosql/tests/ProbabilityTest.cpp:168]: Are the tests from line 168 - 170 needed. ? Lets remove them unless they are testing some critical path in the function.

**Review by aditi-pandit — COMMENTED** [review]
Thanks @rrando901 for this PR. Have one minor comment.

**Review by aditi-pandit — APPROVED** [review]
Thanks for your PR.

**Pull request closed without merging**